bindings/node/binding_test.js

```
@@ -0,0 +1,10 @@
+import assert from "node:assert";
+import { test } from "node:test";
+
+test("can load grammar", () => {
+  assert.doesNotReject(async () => {
+    const { default: binding } = await import("./index.js");
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)